functions/src/firestore-types/lobby.ts

```diff
@@ -24,6 +24,11 @@ export type Player = {
    * the number of players that have voted for this player
    */
   votes?: number;
+
+  /**
+   * the role that the player played; used at the end of the game
+   */
+  role: Role;
```

Comment:
this should probobly be optional

Reply:
Sorry can you expand on this? Is there an optional type in Typescript I'm blanking on?